curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/fill-in-the-blank-with-placeholder-text.english.md

```diff
@@ -8,9 +8,9 @@ forumTopicId: 18178
 
 ## Description
 <section id='description'>
-Web developers traditionally use <code>lorem ipsum text</code> as placeholder text. The 'lorem ipsum' text is randomly scraped from a famous passage by Cicero of Ancient Rome.
+Web developers traditionally use <dfn>lorem ipsum text</dfn> as placeholder text. The 'lorem ipsum' text is randomly scraped from a famous passage by Cicero of Ancient Rome.
 Lorem ipsum text has been used as placeholder text by typesetters since the 16th century, and this tradition continues on the web.
-Well, 5 centuries is long enough. Since we're building a CatPhotoApp, let's use something called <code>kitty ipsum text</code>.
+Well, 5 centuries is long enough. Since we're building a CatPhotoApp, let's use something called <dfn>kitty ipsum text</dfn>.
```

Comment:
I would say either revert back to code tags or have no tags at all.

Reply:
I think it doesn't really make sense to have code tags, as this isn't code. It should probably have no tags as you said, but maybe it should have quotes:
> something called "kitty ipsum text"

Reply:
Yes, change it to "kitty ipsum text", since we are not defining it.